src/test/java/testsuite/failover/NetworkFailuresFailoverIntegrationTest.java

```diff
@@ -90,8 +91,8 @@
   private static final String DB_CONN_CLUSTER_RO = "jdbc:mysql:aws://" + DB_HOST_CLUSTER_RO + "/" + DB_DATABASE;
   private static final String DB_CONN_INSTANCE_PATTERN = "jdbc:mysql:aws://" + DB_HOST_INSTANCE_PATTERN + "/" + DB_DATABASE;
 
-  private static final String DB_USER = System.getProperty("com.mysql.cj.testsuite.failover.networkFailures.user");
```

Comment:
I'd prefer to move towards properties files instead of using environment variables.

Reply:
I would prefer keeping it in environment variables because I don't want to accidentally make the mistake of pushing the property files with the variables in plain text 😅